src/main/java/io/codeka/gaia/hcl/HclParser.kt

```diff
@@ -0,0 +1,39 @@
+package io.codeka.gaia.hcl
+
+import io.codeka.gaia.hcl.antlr.hclLexer
+import io.codeka.gaia.hcl.antlr.hclParser
+import io.codeka.gaia.modules.bo.Output
+import io.codeka.gaia.modules.bo.Variable
+import org.antlr.v4.runtime.CharStreams
+import org.antlr.v4.runtime.CommonTokenStream
+
+class HclParser {
+
+    private fun parseContent(content: String): HclVisitor {
+        // loading test file
+        val charStream = CharStreams.fromString(content)
+
+        // configuring antlr lexer
+        val lexer = hclLexer(charStream)
+
+        // configuring antlr parser
+        val tokenStream = CommonTokenStream(lexer)
+        val parser = hclParser(tokenStream)
+
+        // when
```

Comment:
missing comment?

Reply:
old unit tests comment :laughing: 

comments are not very clear here, I've corrected that